### PR TITLE
Measure code coverage in azure-pipelines.yml

### PR DESCRIPTION
**Pull request by atwoosnam**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -22,15 +28,20 @@ steps:
...
-# Test frameworks that have a Visual Studio test adapter such as MsTest, xUnit, NUnit, Chutzpah (for JavaScript tests using QUnit, Mocha and Jasmine), etc. can be run.
-# Tests can be distributed on multiple agents using this task (version 2).
-- task: VSTest@2
+- task: DotNetCoreCLI@2
+  displayName: Test
+  inputs:
+    command: test
+    projects: |
+      **/test/*.Tests.csproj
+    arguments: '--configuration $(buildConfiguration) /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura /p:Include="[Tanzu.*]*"'
```
Comment (TimHess) [inline azure-pipelines.yml:37]:
```suggestion
    projects: **/test/*.Tests.csproj
    arguments: '--no-build --configuration $(buildConfiguration) --collect "Code coverage"'
```

Try this

File: azure-pipelines.yml [hunk 2]
```diff
@@ -22,16 +28,13 @@ steps:
...
-- task: VSTest@2
+- task: DotNetCoreCLI@2
+  displayName: Test
   inputs:
-    testSelector: 'testAssemblies'
-    testAssemblyVer2: '**\*test*.dll'
-    searchFolder: '$(System.DefaultWorkingDirectory)'
-
+    command: test 
+    projects: **/test/*.Tests.csproj
```
Comment (TimHess) [inline azure-pipelines.yml:35]:
```suggestion
    projects: `**/test/*.Tests.csproj`
```

File: azure-pipelines.yml [hunk 3]
```diff
@@ -22,15 +28,12 @@ steps:
   inputs:
     solution: '**\*.sln'
```
Comment (TimHess) [inline azure-pipelines.yml:29]:
```suggestion
    solution: '**\*.sln' 
    configuration: $(buildConfiguration)
```

File: azure-pipelines.yml [hunk 4]
```diff
@@ -5,6 +5,12 @@ pool:
   vmImage: 'windows-2019'
```
Comment (TimHess) [inline azure-pipelines.yml:6]:
```suggestion
variables:
  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
  DOTNET_CLI_TELEMETRY_OPTOUT: 1
```

File: azure-pipelines.yml [hunk 5]
```diff
@@ -21,18 +29,25 @@ steps:
...
     solution: '**\*.sln' 
+    configuration: $(buildConfiguration)
+
+- task: DotNetCoreCLI@2
+  displayName: Test
+  inputs:
+    command: test 
+    projects: '**/test/*.Tests.csproj'
+    arguments: '--no-build --configuration $(buildConfiguration) /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura'
+    reports: $(Build.SourcesDirectory)/tests/**/coverage.cobertura.xml
```
Comment (TimHess) [inline azure-pipelines.yml:40]:
```suggestion
    arguments: '--no-build --configuration $(buildConfiguration) --collect "XPlat Code Coverage" /p:CoverletOutputFormat=cobertura'
    reports: $(Agent.TempDirectory)/**/coverage.cobertura.xml
```

**Review by zzori-theoriginal — DISMISSED** [review]

**Review by ccheetham — CHANGES_REQUESTED** [review]
Choose one path delimiter style rather than use a mixture.

I.e., either `/` or `\`.

**Review by ccheetham — APPROVED** [review]
lgtm

**Pull request merged**